packages-ts/starknet-gauntlet-ocr2/test/commands/ocr2.test.ts

```diff
@@ -15,10 +15,10 @@ import { Contract } from 'starknet'
 import { BN } from '@chainlink/gauntlet-core/dist/utils'
 
 const signers = [
-  '0x04cc1bfa99e282e434aef2815ca17337a923cd2c61cf0c7de5b326d7a8603730',
-  '0x04cc1bfa99e282e434aef2815ca17337a923cd2c61cf0c7de5b326d7a8603731',
+  'ocr2on_starknet_04cc1bfa99e282e434aef2815ca17337a923cd2c61cf0c7de5b326d7a8603730',
+  'ocr2on_starknet_0x04cc1bfa99e282e434aef2815ca17337a923cd2c61cf0c7de5b326d7a8603731',
   '0x04cc1bfa99e282e434aef2815ca17337a923cd2c61cf0c7de5b326d7a8603732',
```

Comment:
Did you miss this one?

Reply:
actually that was intentional :) I wanted to make sure the prefix logic could handle various types of inputs. I'll add a few comments to the lines to indicate the different formats